Added header

cfbe opened the pull request.

alfbr commented:
Thank you. The purpose of the test-case is described in the folder name. We should probably consider a more generic folder name if this dataset sees more usage. As for now, we keep it as is. The data-file name describes what the dataset is supposed to test, namely NNCs and LGRs, which is also fine, but please include a description of the dataset in the .DATA file header.

cfbe commented:
As Alf asked for, I added a short description in the *.DATA file. In addition I added a folder with eclipse simulation output that should be of use for the ResInsight testing.

alfbr commented:
Excellent, merging. Please add a separate PR if a parser test is warranted.
